var/spack/repos/builtin/packages/duckdb/package.py

```diff
@@ -11,11 +11,17 @@ class Duckdb(CMakePackage):
     """DuckDB is an in-process SQL OLAP Database Management System."""
 
     homepage = "https://duckdb.org"
-    url = "https://github.com/duckdb/duckdb/archive/refs/tags/v0.8.1.tar.gz"
+    url = "https://github.com/duckdb/duckdb/archive/refs/tags/v0.9.2.tar.gz"
     git = "https://github.com/duckdb/duckdb.git"
 
     license("MIT")
+    maintainers("glentner")
 
     version("master", branch="master")
+    version(
+        "0.9.2",
+        sha256="afff7bd925a98dc2af4039b8ab2159b0705cbf5e0ee05d97f7bb8dce5f880dc2",
+        preferred=True,
```

Comment:
No need to add preferred, if the version is the latest according to our sort algorithm.

Reply:
I've also made a pull request to add variants for duckdb extensions and address a version identification issue in the installer here: https://github.com/spack/spack/pull/38922. Happy to share the maintainer responsibility with you @glentner.